serverless preuninstall Error: Cannot find module

<!--
1. If you have a question and not a bug/feature request please ask it at http://forum.serverless.com
2. Please check if an issue already exists so there are no duplicates
3. Check out and follow our Guidelines: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md
4. Fill out the whole template so we have a good overview on the issue
5. Do not remove any section of the template. If something is not applicable leave it empty but leave it in the Issue
6. Please follow the template, otherwise we'll have to ask you to update it
-->

# This is a Bug Report

## Description

It looks like the problem in #3079 isn't solved yet. I get this error:

```
[developer-portal] npm install -g serverless@1.7                      master  ✭

> serverless@1.7.0 preuninstall /usr/local/lib/node_modules/serverless
> node ./scripts/preuninstall.js

module.js:472
    throw err;
    ^

Error: Cannot find module 'fs-extra'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/serverless/lib/classes/Utils.js:7:36)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/serverless/lib/Serverless.js:12:15)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
npm WARN lifecycle serverless@1.7.0~preuninstall: continuing anyway serverless@1.7.0 preuninstall: `node ./scripts/preuninstall.js`
npm WARN lifecycle Exit status 1
```

(It started with module shelljs and I installed few of the dependencies globally before I realized it is probably a bug in Serverless. ;o)

Similar or dependent issues:
* #3079

## Additional Data

* ***Serverless Framework Version you're using***: 1.7
* ***Operating System***: macOS

--- a/package.json
+++ b/package.json
@@ -55,8 +55,7 @@
     "docs": "node scripts/generate-readme.js",
     "simple-integration-test": "jest --maxWorkers=5 simple-suite",
     "complex-integration-test": "jest --maxWorkers=5 integration",
-    "postinstall": "node ./scripts/postinstall.js",
-    "preuninstall": "node ./scripts/preuninstall.js"
+    "postinstall": "node ./scripts/postinstall.js"
   },
   "jest": {
     "testRegex": "(\\.|/)(tests)\\.js$",

